src/platforms/windows/windowsappimageprovider.h

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef WINDOWSAPPIMAGEPROVIDER_H
+#define WINDOWSAPPIMAGEPROVIDER_H
+
+#include <QObject>
+#include <QQuickImageProvider>
+
+
+class WindowsAppImageProvider final : public QQuickImageProvider,
```

Comment:
This class is cool! Wondering if we should filter out the application without an icon. What do you think?

Reply:
Sure!